### DISCUSSION TITLE
Troubleshooting for HyperSubLayers

### DISCUSSION THREAD

**Discussion by jot-ar** [mxstbr/karabiner#12] [Q&A]
Wondering if anyone knows why some of the sublayer commands do not work.
E.g., the cmd for decrementing volume and brightness does work however the cmd for incrementing does not.
Also, all my shell_commands do not execute despite it being in the right syntax.
All sublayers within the app opening command work perfectly, other sublayers, however not at all. 

Appreciate any help with that. 

P.S. Thank you for typing up such a wonderful tool for Karabiner-Elements

**Comment by mxstbr** [comment]
@JLN-Reisinger could you post your full config? That'd help with debugging this!
